Jasmine/Common/Models/TextGrid.swift

```diff
@@ -55,10 +60,8 @@ struct TextGrid {
     /// - Parameters:
     ///   - coord1: the first coordinate
     ///   - coord2: the second coordinate
-    mutating func swap(_ coord1: Coordinate, and coord2: Coordinate) {
-        let temp = grid[coord2.row][coord2.col]
-        grid[coord2.row][coord2.col] = grid[coord1.row][coord1.col]
-        grid[coord1.row][coord1.col] = temp
+    mutating func swap(coord1: Coordinate, coord2: Coordinate) {
+        Swift.swap(&self[coord1], &self[coord2])
```

Comment:
The 2 variables are kind of symmetric though, so that signature seems weird. Maybe remove the argument label for both of them just like Swift's inbuilt swap?